pkgs/by-name/na/nawk/package.nix

```diff
@@ -8,13 +8,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "nawk";
-  version = "20230911";
+  version = "20240311";
 
   src = fetchFromGitHub {
     owner = "onetrueawk";
     repo = "awk";
     rev = "2ndEdition";
```

Comment:
```suggestion
    rev = finalAttrs.version;
```

Reply:
Ohai @yayayayaka 👋😃 Thanks for catching this!